docs/getting_started.rst

```diff
@@ -43,6 +64,21 @@ From here, we can also convert our ``Pauli`` object back to a dense numpy array
 Pauli Strings
 ------------------------
 
+Pauli strings are tensor-product combinations of Pauli matrices. For example, the following is a valid Pauli string:
+
+.. math::
+
+    \mathcal{\hat{P}} = \sigma_x \otimes \sigma_y \otimes \sigma_z
+
+where :math:`\otimes` denotes the tensor or `Kronecker <https://en.wikipedia.org/wiki/Kronecker_product>`_ product, and we can more simply denote by
+
+.. math::
+
+    \mathcal{\hat{P}} = XYZ
+
+A Pauli string of length ``N`` is a tensor product of ``N``
+Pauli matrices. A ``N``-length Pauli String is a :math:`2^N \times 2^N` matrix, so ``XYZ`` is a :math:`8 \times 8` matrix.
+
 In ``fast_pauli``, we represent Pauli strings using the ``PauliString`` class. For example, to construct the Pauli string ``X, Y, Z``, we can do:
```

Comment:
```suggestion
In ``fast_pauli``, we represent Pauli strings using the ``PauliString`` class, which represents this operator sparsely as just an array of `Pauli` objects. For example, to construct the Pauli string ``X\otimesY\otimesZ``, we can do:
```

Reply:
Since we don't bring up operators yet, could we say `represents this tensor product sparsely....`